composer.json

```diff
@@ -12,6 +12,7 @@
         "php": ">= 8.1",
         "ext-mbstring": "*",
         "ext-pcntl": "*",
+        "ext-zip": "*",
```

Comment:
Where is the `zip` extension needed?

Reply:
Extracting some sources failed for me without the zip extension installed. Let me make sure this is actually the case or if it was just a random fail.

Reply:
Currently, only the `unzip` command is used, and PHP's zip extension is not used. It may be a related problem caused by the downloaded file being damaged.

Reply:
Figured it out.  On RHEL based distros, composer automatically installs ext-zip. If you uninstall it, it breaks composer autoloading. It's not a direct dependency of spc though, so I removed it.